modules/ebs_snapshot_explorer/__init__.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+
+module_info = {
+    'name': 'ebs_snapshot_explorer',
+    'author': 'Alexander Morgenstern alexander.morgenstern@rhinosecuritylab.com',
+    'category': 'post_exploitation',
+    'one_liner': 'Loads EBS snapshots and volumes so they are easily accessible.',
+    'description': 'This module will use an EC2 instance to load Elastic Block Store volumes and snapshots in the account and allow the user to access the data.',
+    'services': ['EC2'],
+    'prerequisite_modules': [],
+    'arguments_to_autocomplete': ['--instance', '--region', '--zone'],
+}
+
+parser = argparse.ArgumentParser(add_help=False, description=module_info['description'])
+parser.add_argument(
+    '--instance',
+    required=True,
+    default=None,
```

Comment:
For this and the other two, you don't need/want a default if it is required, as you will never have a default state